lib/publiccloud/utils.pm

```diff
@@ -260,21 +266,9 @@ sub get_ssh_private_key_path {
     return (is_azure() || is_openstack() || get_var('PUBLIC_CLOUD_LTP')) ? "~/.ssh/id_rsa" : '~/.ssh/id_ed25519';
 }
 
-sub prepare_ssh_tunnel {
+sub permit_passwordless {
```

Comment:
I would say that name of the function represent 1% of actions done by it :thinking: 
you can not give some common name to actions which happens within this function . So I would recommend two options : 
1. Just merge this code into `prepare_ssh_tunnel` because you can not argue that this term will cover all this actions 
2. merge this code together with code from `prepare_ssh_tunnel` AND split it into some logical pieces which CAN get some meaningful names

Reply:
> I would say that name of the function represent 1% of actions done by it 🤔

I don't agree. I cannot merge it with `prepare_ssh_tunnel` (I'm doing actually the opposite) as the tunnel preparation (`~/.ssh/config` placement, normal user creation) happens only once. But `permit_passwordless` (allowing `PermitRootLogin` in sshd and placing `/root/.ssh/authorized_keys`) happens twice as `cleanup_cloudinit()` forces the SUT to rerun the initial VM configuration (e.g. sshd is reconfigured and `/root/.ssh/authorized_keys` is replaced)